File: MyCookBook/MyCookBook/Models/Recipe.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace MyCookBook.Models
{
    public class Recipe
    {
        [Key]
        public int RecipeId { get; set; }
        public string Name { get; set; }
        public string Author { get; set; }
        public bool Private { get; set; }
        public string ImageLink { get; set; }
        public int CookTime { get; set; }
        public int CuisineId { get; set; }
        public int[] CathegoryId { get; set; }
        public int Volume { get; set; }
        public List<Ingridient> Ingridients { get; set; }
        public List<CookStep> Steps { get; set; } 
    }
}

File: MyCookBook/MyCookBook/Scripts/Recipe.js

var cookStepModel=
    {
        RecipeId : 0,
        Name :'',
        Author :'',
        Private : false,
        ImageLink :  '',
        CookTime : 0,
        CuisineId : 0,
        Volume: 0,
        CathegoryId: new Array(),
        Ingridients : new Array(),
        Steps : new Array(),
    };
ko.applyBindings(cookStepModel);


function save(){
    $.ajax({
        url: "Create",
        type: "post",
        contentType: "application/json",
        data: ko.mapping.toJSON(cookStepModel)
    })
};